students/10th/soojungsohn/user/models.py

```diff
@@ -9,3 +9,12 @@ class User(models.Model):
     
     class Meta:
         db_table = 'users'
+
+class Follow(models.Model):
+    main_user   = models.ForeignKey(User, on_delete=models.CASCADE, related_name='main_user')
+    sub_user    = models.ForeignKey(User, on_delete=models.CASCADE, related_name='sub_user')  
```

Comment:
from, to 로 naming 하는게 좋을 것 같습니다. 왜냐하면 나중에 following, followers 불러올때 좀더 가독성이 좋을 것 같습니다.
main, sub 로하면 헷갈릴수 있어서 그렇습니다.
```suggestion
    from_user   = models.ForeignKey(User, on_delete=models.CASCADE, related_name='from_user')
    to_user    = models.ForeignKey(User, on_delete=models.CASCADE, related_name='to_user')  
```